source/content/drupal-commandline.md

```diff
@@ -169,18 +169,17 @@ You may want to remove these modules after you launch your site, or use more adv
 1. Enable the modules:
 
   ```bash{promptUser: user}
-  terminus drush $TERMINUS_SITE.dev -- pm-enable devel devel_generate webprofiler -y
+  terminus drush $TERMINUS_SITE.dev -- pm-enable devel devel_generate -y
```

Comment:
webprofiler isn't included in devel anymore and I don't think it is necessary for the doing of this tutorial